arm-machinelearning/2016-05-01-preview/swagger/webservices.json

```diff
@@ -196,24 +199,27 @@
         "tags": [
           "WebServices"
         ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
         "operationId": "WebServices_ListInResourceGroup",
```

Comment:
One last change: Please name this operation: WebServices_List~~In~~**By**ResourceGroup. That is the standard convention. Those libraries which are not following this convention, we are asking them to change this in the next version. Consistent naming pattern across RPs provides a good experience to customers in Azure and they don't have to keep learning these unnecessary things.